processor/common/common.go

```diff
@@ -0,0 +1,340 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
```

Comment:
My suggestions (as metioned [here](https://github.com/open-telemetry/opentelemetry-collector/pull/537#issuecomment-583250365)): either `spanmatcher` if this deals with only matching with spans or perhaps `span` with we want some other funtionality in the same package.

Reply:
Agree

Reply:
moved everything to be in a span package until internal/processor